examples/single_line/inputs_small_refined.json

```diff
@@ -0,0 +1,39 @@
+{
+  "time": 
+  {
+    "Co": 0.125,
+    "start_time": 0.0,
+    "end_time": 0.0015,
+    "total_output_steps": 2,
+    "total_monitor_steps": 10
+  },
+  "space":
+  {
+    "initial_temperature": 300.0,
+    "cell_size": 3e-6,
+    "global_low_corner": [-210e-6, -180e-6, -180e-6],
+    "global_high_corner": [300e-6, 180e-6, 0.0],
+    "ranks_per_dim": [2, 2, 1]
+  },
+  "properties":
+  {
+    "density": 7500.0,
+    "specific_heat": 750.0,
+    "thermal_conductivity": 25.0,
+    "latent_heat": 2e5,
+    "solidus": 1410.0,
+    "liquidus": 1620.0
+  },
+  "source":
+  {
+    "absorption": 0.3,
+    "two_sigma": [60e-6, 60e-6, 60e-6],
+    "scan_path_file": "/Users/65y/Desktop/finch/examples/single_line/scan_path_small.txt"
```

Comment:
This needs to be relative for a general example, but then it will probably break in the intended CI case..

Reply:
No, because I still want it to work locally in most cases. We could install these example files like we do for ExaCA though

Reply:
This example wouldn't be run as part of the CI anyways, it takes too long. I'll just do what we did with the other file and make it a local path for now